hadoop-yarn-project/hadoop-yarn/hadoop-yarn-csi/pom.xml

```diff
@@ -32,6 +32,25 @@
         <os-maven-plugin.version>1.5.0.Final</os-maven-plugin.version>
     </properties>
 
+    <profiles>
```

Comment:
I think the patch makes sense to me. Thanks for working on this.
Question: will openlab be responsible for making netty4 releases? How frequent does it do (netty typically makes one release every month)? if netty developers end up hosting aarm64 artifacts, should we use the official aarm64 artifacts? 

Code: suggest to add a comment in the pom.xml why we use unofficial artifacts here.

Reply:
@jojochuang thanks for your review,
Actually we have do some  efforts to promote aarch64 support in netty community itself, please see: https://github.com/netty/netty-tcnative/pull/517, but because the maintainer of netty is too busy that the PR still in review progress.  We hope Hadoop can fully support aarch64 platform for recent releases, so this is a workaround, once the netty officially support aarch64, we will use the official aarch64 artifacts. It looks like the netty didn't have a fixed release schedule, but about per month have a release published: https://github.com/netty/netty/releases

Thanks, I will add a comment in the pom.xml